version.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://raw.githubusercontent.com/dotnet/Nerdbank.GitVersioning/main/src/NerdBank.GitVersioning/version.schema.json",
-  "version": "1.10-preview",
+  "version": "1.11-preview",
```

Comment:
What's going on here?

Reply:
Ho, that happened when I rebased on top of main. I can revert this